packages/core/src/App/Containers/RealAccountSignup/financial-details-form.js

```diff
@@ -0,0 +1,410 @@
+        default_value: '',
+        supported_in: ['maltainvest'],
+        rules: [['req']],
+    },
+    employment_status: {
+        default_value: '',
+        supported_in: ['maltainvest'],
+        rules: [],
+    },
+    estimated_worth: {
+        default_value: '',
+        supported_in: ['maltainvest'],
+        rules: [['req']],
+    },
+    income_source: {
+        default_value: '',
+        supported_in: ['maltainvest'],
+        rules: [['req']],
+    },
+    net_income: {
+        default_value: '',
+        supported_in: ['maltainvest'],
+        rules: [['req']],
+    },
+    occupation: {
+        default_value: '',
+        supported_in: ['maltainvest'],
+        rules: [['req']],
+    },
+    other_instruments_trading_experience: {
+        default_value: '',
+        supported_in: ['maltainvest'],
+        rules: [['req']],
+    },
+    other_instruments_trading_frequency: {
+        default_value: '',
+        supported_in: ['maltainvest'],
+        rules: [['req']],
+    },
+    source_of_wealth: {
+        default_value: '',
+        supported_in: ['maltainvest'],
+        rules: [],
+    },
+};
+
+export const financialDetailsConfig = ({ real_account_signup_target }) => {
+    return {
+        header: {
+            active_title: localize('Complete your financial assessment'),
+            title: localize('Financial assessment'),
+        },
+        body: FinancialDetails,
+        form_value: getDefaultFields(real_account_signup_target, financial_details_config),
+        props: {
+            validate: generateValidationFunction(real_account_signup_target, financial_details_config),
+            account_turnover_enum: account_turnover_enum(),
+            binary_options_trading_experience_enum: binary_options_trading_experience_enum(),
+            binary_options_trading_frequency_enum: binary_options_trading_frequency_enum(),
+            cfd_trading_experience_enum: cfd_trading_experience_enum(),
+            cfd_trading_frequency_enum: cfd_trading_frequency_enum(),
+            education_level_enum: education_level_enum(),
+            employment_industry_enum: employment_industry_enum(),
+            employment_status_enum: employment_status_enum(),
+            forex_trading_experience_enum: forex_trading_experience_enum(),
+            forex_trading_frequency_enum: forex_trading_frequency_enum(),
+            estimated_worth_enum: estimated_worth_enum(),
+            income_source_enum: income_source_enum(),
+            net_income_enum: net_income_enum(),
+            occupation_enum: occupation_enum(),
+            other_instruments_trading_experience_enum: other_instruments_trading_experience_enum(),
+            other_instruments_trading_frequency_enum: other_instruments_trading_frequency_enum(),
+            source_of_wealth_enum: source_of_wealth_enum(),
+        },
+        passthrough: ['residence_list', 'is_fully_authenticated'],
+    };
+};
+const account_turnover_enum = () => [
```

Comment:
[Nitpick/discussion] These seem to have a suffix of `enum`, however, they are non-constant array objects. Do you think we should reserve that keyword for when we `Object.freeze`?